File: 10 - Python/Exercícios/05 - Matrizes/Exercício 02.py
#Exercício - Soma Linhas

linha = int(input("Qual a quantidade de linhas da matriz? "))
coluna = int(input("Qual a quantidade de colunas da matriz? "))

mat = [[0 for x in range(coluna)] for x in range(linha)]
vet = [0] * linha

for i in range(linha):
    print(f"Digite os elementos da {i+1}º linha:")
    for j in range(coluna):
        mat[i][j] = float(input(f"Elemento [{i},{j}]: "))

print("")
print("Vetor gerado:")

for i in range(linha):
    vet[i] = 0
    for j in range(coluna): 
        vet[i] = vet[i] + mat[i][j]
    print(vet[i])

File: 07 - C++/Exercícios/05 - Matrizes/Exercício 02.cpp
//Exercício - Soma Linhas

#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    int i, j, linha, coluna;

    cout << "Qual a quantidade de linhas da matriz? ";
    cin >> linha;
    cout << "Qual a quantidade de colunas da matriz? ";
    cin >> coluna;

    double mat[linha][coluna], vet[linha];

    for(i = 0; i < linha; i++){
        cout << endl << "Digite os elementos da " << i+1 << "a linha: " << endl;
        for(j = 0; j < coluna; j++){
            cin >> mat[i][j];
        }
    }

    cout << fixed << setprecision(1);
    cout << endl << "Vetor gerado:" << endl;

    for(i = 0; i < linha; i++){
        vet[i] = 0;
        for(j = 0; j < coluna; j++){
            vet[i] = vet[i] + mat[i][j];
        }
        cout << vet[i] << endl;
    }

    return 0;

}